<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QualityApply</class>
 <widget class="QWidget" name="QualityApply">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1700</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1700</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_left_bottom" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>126</y>
     <width>441</width>
     <height>625</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>441</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>441</width>
     <height>650</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_left_bottom{
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_top_left" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>441</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>441</width>
     <height>115</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>441</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_top_left{
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_right_bottom" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>126</y>
     <width>1230</width>
     <height>625</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1230</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>650</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_right_bottom
{
background: #FFFFFF;
border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_top_right" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>1230</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1230</width>
     <height>110</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>130</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_top_right
{
	border-radius: 10px;
	background: #FFFFFF;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
